features/networking/operator.feature

```diff
@@ -11,7 +11,5 @@ Feature: Operator related networking scenarios
     And evaluation of `cluster_operator('network').condition(type: 'Available')` is stored in the :operator_status clipboard
     #Making sure that network operator AVAILABLE status value is True
     Then the expression should be true> cb.operator_status["status"]=="True"
-
-    Given evaluation of `cluster_operator('network').versions` is stored in the :operator_version clipboard
-    #Confirm whether network operator version matches with ocp version`
-    Then the expression should be true> cb.operator_version=cb.ocp_version
+    #Confirm whether network operator version matches with ocp version
+    And the expression should be true> cluster_operator('network').version_exists?(version: cb.ocp_version)==true
```

Comment:
@pruan-rht `cluster_operator('network').versions` is not present in latest commit as versions_exists def suffice that check. Regarding `And the expression should be true> cluster_operator('network').version_exists?(version: cb.ocp_version)`, i believe i verified that guess i had some check issues. I might be wrong but will check again

Reply:
@anuragthehatter both versions work, but `cluster_operator('network').version_exists?(version: cb.ocp_version)` is enough since the `#version_exists?` returns a boolean already.

Reply:
Yep, confirmed again. `==true` is unnecessary, removed that. Thanks!

Reply:
@anuragthehatter thanks for the update...I will merge it now.

Reply:
Agree, Peter. `.versions` returns version in an array while version_exists? returns boolean. Both would be useful based on what the usecase is.